src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/reference/inputs/EceCharacterReferenceEvaluationPreview.vue

```diff
@@ -5,31 +5,31 @@
         <v-col cols="4">
           <p class="small">What is your relationship with the applicant?</p>
         </v-col>
-        <v-col>
+        <v-col cols="8">
           <p class="small font-weight-bold">{{ evaluation.referenceRelationship }}</p>
         </v-col>
       </v-row>
       <v-row v-if="evaluation.referenceRelationship === 'Other'">
         <v-col cols="4">
           <p class="small">Specify your relationship with the applicant</p>
         </v-col>
-        <v-col>
+        <v-col cols="8">
           <p class="small font-weight-bold">{{ evaluation.referenceRelationshipOther }}</p>
         </v-col>
       </v-row>
       <v-row>
         <v-col cols="4">
           <p class="small">How long have you known the applicant?</p>
         </v-col>
-        <v-col>
+        <v-col cols="8">
           <p class="small font-weight-bold">{{ evaluation.lengthOfAcquaintance }}</p>
         </v-col>
       </v-row>
       <v-row>
         <v-col cols="4">
```

Comment:
Do these cols style well on mobile? Usually I find 4 and even 8 is tight on mobile where we want to go to 10+...

Reply:
Styles OK on mobile:

![image](https://github.com/bcgov/ECC-ECER/assets/160776397/1e537ed4-6834-40c2-896e-30828c916e24)